### PR TITLE
Merge 19.8 code freeze

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

File: WordPress/Resources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -7167,12 +7220,39 @@
...
+"stats.insights.latestPostSummary.publishDate" = "Published %@";
+
+/* Title for Views count in Latest Post Summary stats card. */
+"stats.insights.latestPostSummary.views" = "Views";
+
+/* Insights 'Most Popular Time' header. Fire emoji should remain part of the string. */
+"stats.insights.mostPopularCard.title" = "🔥 Most Popular Time";
+
+/* Label showing the percentage of views to a user's site which fall on a particular day. */
+"stats.insights.mostPopularCard.viewPercentage" = "%d%% of views";
```
Comment (mokagio) [inline WordPress/Resources/en.lproj/Localizable.strings:7239]: Love the reverse DNS localization keys 👍

**Review by mokagio — COMMENTED** [review]
- Switch to production Pods – None required in this release
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig` and `download_metadata.swift`

File: WordPress/Resources/en.lproj/Localizable.strings (hunk 1 above)
Comment (AliSoftware) [reply WordPress/Resources/en.lproj/Localizable.strings:7239]: whoop whoop

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 2]
```diff
@@ -1,11 +1,13 @@
-We made a small but mighty change to the block editor: the “Add Block” button is more visible when you first open the editor with no blocks selected. We also removed one of three error notifications when a media or audio block fails to upload—it felt like overkill.
+* [**] Self hosted sites are not restricted by video length during media uploads [https://github.com/wordpress-mobile/WordPress-iOS/pull/18414]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: Items about self-hosted site should be removed from Jetpack Release Notes, right @leandroalonso ?

**Review by AliSoftware — CHANGES_REQUESTED** [review]
- [ ] Pods updated to stable version
    - ❓  @mokagio seems there were no new Pods to release this time… is that expected? (maybe it is, but just wanted to double-check)
 - [x] New header in `RELEASE-NOTES.txt`
 - [x] JP and WP `release_notes.txt` updated with draft notes
 - [x] JP release notes audited for differences between WP and JP
    - ⚠️  @mokagio @leandroalonso I think Jetpack release notes will need to be adjusted?
 - [x] `WordPress/Resources/en.lproj/Localizable.strings` updated with latest strings
 - [x] Version bump in `.xcconfig` files

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 2 above)
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt]: @AliSoftware yep!

Comment (AliSoftware) [reply WordPress/Jetpack/Resources/release_notes.txt]:
```suggestion
```

Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt]: Dang! I missed that... Thank you for noticing and confirming @AliSoftware @leandroalonso

**Comment by mokagio** [comment]
@AliSoftware 

>  Pods updated to stable version
> ❓ @mokagio seems there were no new Pods to release this time… is that expected? (maybe it is, but just wanted to double-check)

I did mark it as expected in my PR review

> Switch to production Pods – None required in this release

~~However, I was sure I left a note more detailed note about this but I can't find it anywhere. 🤔~~  Oh! Here it is, just a matter of browser cache on my end p8Qyks-2I6-p2#comment-1055

**Comment by mokagio** [comment]
Turns out there should have been a `Podfile` update, but both my eyes and our tooling missed it. See [`4a4126a`](https://github.com/wordpress-mobile/WordPress-iOS/pull/18486/commits/4a4126a1f1ad2df889a34539983327779909a5f2).

That commit also generates a conflict with `trunk`, so I'll have to close this PR and open a new one from a merge resolution branch.

**Comment by AliSoftware** [comment]
> I did mark it as expected in my PR review

Doh! I have to admit I didn't read that review with as much attention as I usually read PR _descriptions_.

Do you think in the future, instead of leaving empty PR description and providing a review as separate comment, you could provide your checklist directly in the PR description instead? 🙏

File: Podfile.lock [hunk 3]
```diff
@@ -487,7 +487,7 @@ PODS:
     - WordPressKit (~> 4.18-beta)
     - WordPressShared (~> 1.12-beta)
     - WordPressUI (~> 1.7-beta)
-  - WordPressKit (4.51.0-beta.1):
+  - WordPressKit (4.51.0):
```
Comment (AliSoftware) [inline Podfile.lock:490]: This is exactly why I opened https://github.com/wordpress-mobile/release-toolkit/issues/216 a while ago (wow… more than a year ago now 😓 ). I never found time to get to it, but maybe this is a good reminder that we should tackle this sooner than later?

**Comment by mokagio** [comment]
> Do you think in the future, instead of leaving empty PR description and providing a review as separate comment, you could provide your checklist directly in the PR description instead? 🙏

Sure 😄

I like using the review because it sort of tracks via the GitHub UI that I reviewed the diff (or at least went in the `/files` tab and pressed Review > Comment 😅 ))

But, I do find the comment it posts is easy to miss among all the other things that get posted in the PR page. 😳

I'll experiment with posting the checklist in the description for visibility and doing something else with "Review > Comment" to tame my process anxiety 🙃 👍

**Comment by AliSoftware** [comment]
For the record / your info, the way I tend to do it personally:
 - Checklist in PR description: 2 options
   1. Either I create the PR as a Draft, with the checklists with _unchecked_ items as the PR description, then review the PR's diff (in the Files tab) for each item in my checklist once the PR is created, and finally I check the boxes in the draft PR description to update it accordingly (no need to enter "edit" mode, you can click on the boxes directly to check/uncheck them)
   2. Or, while I'm still on the UI where I'm about to create the PR but haven't created it yet, I scroll down the page to see the diff of the PR I'm about to create, do a quick check that the expected files are here, and check the boxes in my checklist directly. Then I create the PR (as a ready-for-review, non-draft) with that already-ready description
 - Then, I do a self-review my own PR
   - Either I have some additional in-line comments to add about specific changes or lines, to explain why they're here etc., so I add them and submit a self-review
   - Or I have nothing to add during my self-review, so I don't publish any review comment
 - Once I've self-reviewed my PR, I undraft it (if I created it as Draft), then add the owl-team as reviewers.

I had been doing option (i) for a while, but nowadays I tend to do option (ii) — i.e. go thru the checklist before I even validate the PR creation, rather than create as Draft and edit description afterwards — because (ii) avoids switching back and forth between the "Conversation" and "Files Changed" tab (when I was using (i), I often opened 2 Google Chrome tabs on the PR — one on each of the GitHub tabs of that PR — to allow me easy back-and-forth when going thru the checklist… no more with (ii))

**Comment by AliSoftware** [comment]
Alternatively, if you prefer to keep your process of posting the checklist as a PR review, maybe instead of leaving the PR description empty, edit it to indicate something like "Code freeze of 19.8; I've self-reviewed the changes in [link to your self-review]"? 🤔 

idk, maybe it's just me who should also be more attentive at the self-reviews / comments posted in the PR after all.

But PRs with a completely empty description (even for PRs where you think it should be obvious from the PR title or similar) tend to unconsciously make me think "the PR author didn't take much time crafting this as a nice PR, they probably just created it in a hurry without taking time in writing a good description and self-reviewing it"… which is ironic because in your case it's totally the opposite (as you do self-reviews — which I greatly appreciate, btw! — and take the time to self-check your PRs)… but idk, it's just a feel that empty PR descriptions leave in my mouth and I can't help it 😅

**Pull request closed without merging**